JS/fluentRevealNavbar.uc.js

```diff
@@ -149,7 +184,9 @@
         area = el.querySelector(".toolbarbutton-text");
       }
 
-      if (el.disabled || areaStyle.pointerEvents == "none") {
+      // previous pointerEvents check may break the effect in FF 115
+      //if (el.disabled || areaStyle.pointerEvents == "none") {
+      if (el.disabled) {
```

Comment:
I just checked and it works with this check for me

Reply:
Lol I messed this up too. It should say `getComputedStyle(el).pointerEvents == "none"`.
`el.pointerEvents` is always gonna be undefined since `el` is an element, not a CSS2Properties object. My mistake.

Reply:
But I guess, this would revert the check to the original form, as it would be basically the same as `areaStyle = getComputedStyle(area)`? Which in my case resulted in script not working, as it always returned "true"

Reply:
No. The reason it was breaking the script is because `area` is not a `toolbarbutton` but a child of the `toolbarbutton`. [See this code?](https://searchfox.org/mozilla-central/rev/920be8b5eee004fc10a2785fab49b860be4d4ba3/toolkit/content/xul.css#237-239) It's making the children of the button unclickable. But the button itself is clickable. We don't want to show this effect on unclickable elements, just like Firefox doesn't show backgrounds when hovering unclickable elements. But `areaStyle` doesn't represent the style of the actual element, it represents the style of the child element we're displaying the effect on. We can't display the effect on the actual element because it has extra invisible padding. The normal background Firefox displays on buttons is actually on the `.toolbarbutton-icon` or `.toolbarbutton-badge-stack`, so that's where we place our effect, but if the element was disabled, it would be the button itself that would have `pointer-events: none`, not the icon/badge-stack.

Reply:
Anyway, you can just test it yourself. I already tried it earlier today with no issues.

Reply:
Okay, got it!